cpp/test/cpu_operations_test/multiply_test.cc

```diff
@@ -1,53 +1,42 @@
-#include <iostream>
+// Copyright 2016 <Morgan Rockett>
```

Comment:
This should be the MIT License, you can copy and paste the license from src/core/cpu_operations.cc